web-console/future-openshift/nodes/nodes.md

```diff
@@ -0,0 +1,12 @@
+# Nodes
+
+## Compact List View
+![1](img/nodes.png)
+- The compact list view should include Node Name, Status, Role, and Machine
+
+## Expanded List View
+![2](img/nodes-expanded.png)
```

Comment:
@beanh66 do you think you should include a mockup of the responsive behavior for the expanded?

Reply:
Added mobile view @serenamarie125 👍